exporter/exporterhelper/queued_retry_inmemory.go

```diff
@@ -43,6 +40,12 @@ type QueueSettings struct {
 	NumConsumers int `mapstructure:"num_consumers"`
 	// QueueSize is the maximum number of batches allowed in queue at a given time.
 	QueueSize int `mapstructure:"queue_size"`
+	// StorageID if not empty, enables the persistent storage and uses the component specified
+	// as a storage extension for the persistent queue
+	StorageID *config.ComponentID `mapstructure:"storage"`
+	// StorageEnabled describes whether persistence via a file storage extension is enabled using the single
+	// default storage extension.
+	StorageEnabled bool `mapstructure:"persistent_storage_enabled"`
```

Comment:
What happens is `storage` is specified and `persistent_storage_enabled` is also true? Which setting wins?

Reply:
I think the more trickier example is `persistent_storage_enabled: false` and `storage` referring some existing storage extension. Basing on the name, I would assume `persistent_storage_enabled` should have precedence